src/ResourceManagement/Automation/AutomationManagement/Microsoft.Azure.Management.Automation.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.Automation
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.Automation">
-      <PackageVersion>2.0.2</PackageVersion>
+      <PackageVersion>2.0.4</PackageVersion>
```

Comment:
@safeermohammed added a comment to the earlier commit - "Nuget already seems to have 2.0.3. Make it 2.0.4 if this is conflicting". I verified in the Nuget package manager to verify that a 2.0.3 version was published for the Automation.dll, which was released on 3/23/2017. We couldn't find out how was this released (if the code says 2.0.2 is the latest). Thus, to be on a safer side, we bumped it to 2.0.4.